cost/aws/gp3_volume_upgrade/aws_upgrade_to_gp3_volume.pt

```diff
@@ -569,12 +629,24 @@ EOS
       field "size" do
         label "Size"
       end
+      field "iops" do
+        label "IOPS"
+      end
       field "tags" do
         label "Tags"
       end
       field "savings" do
```

Comment:
Since we only show 6 fields by default, I think we need to move some of these up.